concepts/80 Troubleshooting/30 3rd-Party Frameworks Issues/00 Bootstrap/05 Modals.md

```diff
@@ -0,0 +1,68 @@
+If you place DevExtreme components in the [Bootstrap modal](https://getbootstrap.com/docs/5.3/components/modal/), the following issues may occur:
```

Comment:
I would change the article. We can create many "modals" and place DevExtreme components in any of them. So "in a Bootstrap modal" seems a better choice. This also matches some of the uses from their documentation. 

> Popover in a modal
> Tooltips in a modal


```suggestion
If you place DevExtreme components in a [Bootstrap modal](https://getbootstrap.com/docs/5.3/components/modal/), the following issues may occur:
```